src/databricks/labs/ucx/install.py

```diff
@@ -343,6 +345,34 @@ def _configure_inventory_database(self):
                     raise SystemExit(msg)
         return inventory_database
 
+    def _valid_cluster_id(self, cluster_id: str) -> bool:
+        """Lite validation of user supplied cluster id"""
+        return CLUSTER_ID_LENGTH == len(cluster_id)
+
+    def _configure_override_clusters(self):
+        """User may override standard job clusters with interactive clusters"""
+        default_val = ""
+        preamble = "Pick default and we'll create a compatible job cluster (recommended) or "
+        cluster_id = self._question(
```

Comment:
Use _prompt and present the user with a list of interactive clusters (maybe up to 20)